skimage/transform/hough_transform.py

```diff
@@ -125,17 +64,16 @@ def hough_line_peaks(hspace, angles, dists, min_distance=9, min_angle=10,
 
     distance_size = 2 * min_distance + 1
     angle_size = 2 * min_angle + 1
-    hspace_max = ndi.maximum_filter1d(hspace, size=distance_size, axis=0,
-                                      mode='constant', cval=0)
-    hspace_max = ndi.maximum_filter1d(hspace_max, size=angle_size, axis=1,
-                                      mode='constant', cval=0)
+    hspace_max = ndimage.maximum_filter1d(hspace, size=distance_size, axis=0,
+                                          mode='constant', cval=0)
+    hspace_max = ndimage.maximum_filter1d(hspace_max, size=angle_size, axis=1,
+                                          mode='constant', cval=0)
```

Comment:
I wonder why not just to use nd version of `maximum_filter`? (Not a blocker for this PR.)